services/ui-src/serverless.yml

```diff
@@ -48,5 +48,6 @@ custom:
         export COGNITO_CLIENT_ID=${self:custom.cognito_client_id}
         export S3_ATTACHMENTS_BUCKET_REGION=${self:custom.s3_attachments_bucket_region}
         export S3_ATTACHMENTS_BUCKET_NAME=${self:custom.s3_attachments_bucket_name}
+        export LOCAL_LOGIN=false
```

Comment:
I believe if you drop the local defaults from ui-src/.env, you won't need this here.